.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+### Instructions
+
+- Use a **concise** yet **descriptive** title;
```

Comment:
Perhaps all commits should have a title, and that title is the generally same as the PR.  See: http://chris.beams.io/posts/git-commit/